.github/workflows/verify-prod.yaml

```diff
@@ -0,0 +1,29 @@
+# Copyright 2024 Chainguard, Inc.
+# SPDX-License-Identifier: Apache-2.0
+
+name: Verify prod Octo-sts
+
+on:
+  pull_request_target:
+    branches:
+      - 'main'
```

Comment:
I think we want to add a labeling requirement here as well, since this repo is now public.

Reply:
oh, ok, so we will need to trigger when is updated as well

Reply:
are we doing a similar thing in another repo? or will this be the first one?

Reply:
we should, but not yet